Sources/PostmarkClient/Provider.swift

```diff
@@ -0,0 +1,47 @@
+import Vapor
+import HTTP
+
+
+public final class Provider: Vapor.Provider {
```

Comment:
This class was blatantly copied from `SendGridClient`'s  provider. It's too magical for my taste, there are lots of things going on that I'm not sure how or why. I hope to get this cleaned up later on.